exporters/otlp/otlptrace/README.md

```diff
@@ -0,0 +1,35 @@
+# OpenTelemetry-Go OTLP Span Exporter
+
+[![Go Reference](https://pkg.go.dev/badge/go.opentelemetry.io/otel/exporters/otlp/otlptrace.svg)](https://pkg.go.dev/go.opentelemetry.io/otel/exporters/otlp/otlptrace)
+
+OpenTelemetry Protocol (OTLP) Span Exporter.
+
+## Installation
+
+```
+go get -u go.opentelemetry.io/otel/exporters/otlp/otlptrace
+```
+
+## Example
+
+To constructs a new OTLP trace Exporter, you can follow this
+[`OTLP Trace Exporter Example`](https://github.com/open-telemetry/opentelemetry-go/blob/main/exporters/otlp/otlptrace/example_test.go).
+
+Also,if you are not familiar with how to export trace and metric data from the OpenTelemetry-Go SDK to
+the OpenTelemetry Collector, you can reference this
+[`OpenTelemetry Collector Traces Example`](https://github.com/open-telemetry/opentelemetry-go/tree/main/example/otel-collector).
```

Comment:
The title is wrong as it also describes metrics. I suggest just putting hyperlinks to the examples. Easier to have everything in sync.

Reply:
I am lazy and I am suggesting https://github.com/open-telemetry/opentelemetry-go/pull/2053/files#r679920638 😉